robot/src/main/java/us/ilite/robot/commands/TargetLock.java

```diff
@@ -46,7 +48,8 @@ public TargetLock(Drive pDrive, double pAllowableError, SearchDirection pCubeSea
 
     @Override
     public void init(double pNow) {
-        mPID = new PIDController(kP, kI, kD);
+        PIDGains PIDGains = new PIDGains(kP, kI, kD, 0);
```

Comment:
Update to use the PidController

Reply:
Sorry, I'm a little confused as to what you mean. I think it may have updated on it's own after I merged Master into my branch again. ```
@Override
    public void init(double pNow) {
        PIDGains PIDGains = new PIDGains(kP, kI, kD, 0);
        mPID = new PIDController(PIDGains, SystemSettings.kControlLoopPeriod);
```